pom.xml

```diff
@@ -545,6 +545,8 @@
         <version.org.jboss.spec.jakarta.el.jboss-el-api_5.0_spec>4.0.1.Final</version.org.jboss.spec.jakarta.el.jboss-el-api_5.0_spec>
         <version.org.jboss.spec.jakarta.xml.soap.saaj-api_3.0_spec>1.0.0.Final</version.org.jboss.spec.jakarta.xml.soap.saaj-api_3.0_spec>
         <version.org.jboss.spec.jakarta.xml.ws.api_4.0_spec>1.0.0.Final</version.org.jboss.spec.jakarta.xml.ws.api_4.0_spec>
+        <version.org.jboss.universe.community>1.2.0.Final</version.org.jboss.universe.community>
+        <version.org.jboss.universe.producer.wildfly>1.3.11.Final</version.org.jboss.universe.producer.wildfly>
```

Comment:
I'm not sure if that matters, but are the producer artifacts updated before or after WildFly release? 

For example before WildFly 33.0.0.Final is released, do we need to release the producer artifact including the Galleon channel `33.0` so that we can include it in the `wildfly` channel?

Reply:
@spyrkob We release the producer artifact before the relevant Beta. Ideally well before.

https://github.com/wildfly/wildfly-producers/blob/1.3.11.Final/pom.xml with the WF 34 entries was released on July 23.

Reply:
If this change means we'll need to do them even sooner, i.e. adding the WF 35 entries before moving main to WF 35 work after the 34.0.0.Final tag, please let me know.

It's certainly doable; I could do it for 35 today. It's just a matter of knowing when it must be done, which is currently before we release the beta.

Reply:
OK great thank you. 👍